core/ui/compose/testing/src/main/kotlin/app/k9mail/core/ui/compose/testing/ComposeTest.kt

```diff
@@ -31,8 +33,22 @@ open class ComposeTest {
     }
 }
 
+/**
+ * Set the content of the test
+ */
 fun ComposeTest.setContent(content: @Composable () -> Unit) = composeTestRule.setContent(content)
 
+/**
+ * Set the content of the test and wrap it in the thunderbird theme
```

Comment:
```suggestion
 * Set the content of the test and wrap it in the K-9 Mail theme.
```

Reply:
Yeah, the Thunderbird theme is not available yet, but I did set it as the default. Will rename this.